flashcards/backend/decks/tests/test_flashcard_model.py

```diff
@@ -0,0 +1,42 @@
+from datetime import date
+
+from django.contrib.contenttypes.models import ContentType
+from django.utils import timezone
+
+
+def test_correct_gui_representation(flashcard, remove_test_data):
+    assert str(flashcard) == "Test Flashcard"
+
+
+def test_flashcard_creation(deck, user, remove_test_data, difficulty_level, flashcard, category, text):
+    assert flashcard.name == 'Test Flashcard'
+    assert flashcard.deck == deck
+    assert flashcard.category == category
+    assert flashcard.is_active == True

```

Comment:
you are breaking PEP-8 rule. It should be `assert flashcard.is_active` since value of this field is either `True` or `False` it is enough to assert only it's value.